coalib/results/Diff.py

```diff
@@ -375,12 +375,25 @@ def add_lines(self, line_nr_before, lines):
 
         linediff = self._get_change(line_nr_before, min_line=0)
         if linediff.add_after is not False:
-            raise ConflictError('Cannot add lines after the given line since '
-                                'there are already lines.')
+            raise ConflictError('Cannot add line(s) after the given line since '
+                                'there are already line(s).')
```

Comment:
even when using `add_line`, the message makes sense when it stays like before :)